docs/Radarr/Radarr-recommended-naming-scheme.md

```diff
@@ -121,70 +170,84 @@ The filename can be Obscured where the Release naming isn't, especially when you
 
 ## Movie Folder Format
 
-!!! danger ""
+!!! warning "Please note that folder names are created in the database whenever the movie is added to Radarr, and it may be missing or incorrect at that time, resulting in your folder having a blank ID.:bangbang:<br><br>If you instead add it to the filename, the IMDb/TMDb ID will be freshly pulled for any download or upgrade."
```

Comment:
```suggestion
!!! warning "Please note that folder names are created in the database whenever the movie is added to Radarr and it may be missing or incorrect at that time, resulting in your folder having a blank ID.:bangbang:<br><br>If you instead add it to the filename, the IMDb/TMDb ID will be freshly pulled for any download or upgrade."